common.ts

```diff
@@ -5,6 +5,12 @@ import { JsonRpcProvider, Provider } from '@ethersproject/providers'
 chai.use(chaiAsPromised)
 const should = chai.should()
 
+if (process.env.NODE_ENV == 'local') {
+  // Load up env variables
+  var envPath = __dirname + "/.env"
```

Comment:
Can use `const` here as well as https://nodejs.org/api/path.html#path_path_join_paths so its more likely to work on windows

Reply:
Good catch